Directory.Build.props

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project>
-  <PropertyGroup>
-    <ImportNetSdkFromRepoToolset>false</ImportNetSdkFromRepoToolset>
-    
-    <PublishWindowsPdb>true</PublishWindowsPdb>
-  </PropertyGroup>
-
-  <Import Project="$(MsBuildThisFileDirectory)SystemResources.props"/>
   <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
```

Comment:
I don't recall whether SystemResources.props really needs to come before Arcade/Sdk.props  - perhaps we don't need to reorder the props just now.

Reply:
The Arcade [recommendation](https://github.com/dotnet/arcade/blob/master/Documentation/ArcadeSdk.md#directorybuildprops) is to import Sdk.props first:

> The Sdk.props file sets various properties and item groups to default values. It is recommended to perform any customizations after importing the SDK.

Should I file an issue to do so instead?

Reply:
Since SystemResporces.props only defines EmbeddedResource ItemDefinitionGroup, I don't see why  it would need to be before Sdk.props import.

Reply:
Looking through the binlogs from the CI build, its clear that the change  has no adverse effect - you are absolutely right.